docs/src/components/community-guides.tsx

```diff
@@ -33,6 +33,11 @@ const guides: CommunityGuidesProps[] = [
     description: 'Documentation for simple podman setup using quadlets.',
     url: 'https://github.com/tbelway/immich-podman-quadlets/blob/main/docs/install/podman-quadlet.md',
   },
+  {
+    title: 'gphotos import + albums',
+    description: 'Import your G photos files into Immich and add your albums',
```

Comment:
I would prefer writing either "GPhotos" or "Google Photos". I think "Google Photos" would be the best, but I'm open for other opinions :)

Reply:
Agreed, I wasn't sure how problematic it would be to write the word Google in our documentation, but I guess that's fine.